src/Helper/CkoLogger.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Checkoutcom\Helper;
+
+use Checkoutcom\Config\Config;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Client;
+use RuntimeException;
+use Checkoutcom\Helper\Utilities;
+use Psr\Log\LoggerInterface;
+
+class CkoLogger {
+
+    public const SOURCE = '/shopware6';
+    public const SPECVERSION = '1.0';
+
+    /**
+     * @var LoggerInterface
+     */
+    public static $logger;
+
+    function __construct(LoggerInterface $logger) {
+        self::$logger = $logger;
+    }
+    
+    /**
+     * construct logging body for cloudEvent API
+     * post request
+     */
+    private function postLoggingRequest($data) {
+    
+        $logBody = [];
+    
+        $logBody['specversion'] = self::SPECVERSION;
+        $logBody['id'] = $data['id'];
+        $logBody['type'] = $data['type'];
+        $logBody['source'] = self::SOURCE;
+        $logBody['data']['scope'] = $data['scope'];
+        $logBody['data']['message'] = $data['message'];
+        $logBody['cko']['correlationId'] = $data['id'];
+        $logBody['cko']['loglevel'] = 'info';
+    
+        $header =  [
+            'Content-Type' => 'application/cloudevents+json',
+        ];
+    
+        $url = Url::getCloudEventUrl(config::publicKey());
```

Comment:
We don't need to branch based on the current environment, we can just alwayys post to production API (for cloudevents) and split the traffic based on `source` attribute. Requirement is to be able to zoom into a specific Merchant (store instance).

![Screenshot 2021-01-06 at 08 45 37](https://user-images.githubusercontent.com/64793622/103748172-9d565b00-4ffb-11eb-8562-38032c209daf.png)